libvirt/CONTRIBUTING.md

```diff
@@ -15,35 +15,42 @@ Under the hood Virt Stats relies on awesome [libvirt](https://libvirt.org/).
     ```
 - Download a [Debian Cloud image](https://cloud.debian.org/images/cloud/) suitable for QEMU, for example:
     ```
-    $ curl -L -O https://cloud.debian.org/images/cloud/bullseye/latest/debian-11-nocloud-amd64.qcow2
+    $ curl -L -O https://cloud.debian.org/images/cloud/buster/latest/debian-10-nocloud-amd64.qcow2
```

Comment:
I'm curious to understand the idea of having these instructions inside the `CONTRIBUTING.md` vs, the Readme? @przmv Can you please share your motivation/ideas?

Reply:
Because these are the instructions on how to create a minimal lab environment mainly for Virt-Stats development purposes.

Reply:
I see what you mean. I would then add a link/make reference on the Readme to link them up.

Reply:
Done